package.json

```diff
@@ -37,6 +37,7 @@
     "install-peers-cli": "^2.2.0",
     "jest-cli": "^26.0.1",
     "prettier": "^2.0.5",
+    "react-native": "^0.62.2",
```

Comment:
I've added RN as a dev dependency. I'm not actually sure what the right approach is here, because `react-dom` and `react-native` are optional dependencies (one of them is required).

A good approach might be to remove `react-dom` from the peer deps, too.

Reply:
It should work fine to make `react-dom` as a devDependency for the test runner and leave out `react-native` entirely